cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -611,7 +622,7 @@ declare namespace Deno {
    */
   export function dlopen<S extends ForeignLibraryInterface>(
     filename: string | URL,
-    symbols: S,
+    symbols: Const<S>,
   ): DynamicLibrary<S>;
```

Comment:
Note that you get the same hints with just marking the declaration object `as const`.

Reply:
Yeah, that's how I read it as well. It's neat of course :)

Reply:
This trick often used in wasm/napi, because as const is redundant